modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -35,14 +35,16 @@ https://help.salesforce.com/articleView?id=000336819&type=1&mode=1[Run your Sale
 
 == Enable Communities Inside Your Organization
 
-Follow the steps in
+Using the domain name you selected in the previous step, follow the steps in
 https://help.salesforce.com/articleView?id=networks_enable.htm&type=5[Enable Salesforce Communities, role=external, window=blank]
 to enable communities inside your Salesforce organization.
 
 == Deploy Your Domain
 
 Wait to receive the notification email that your domain is registered, and then continue with these steps.
 
+. Log in into your ACM https://login.salesforce.com/[Salesforce Organization, role=external, window=blank].
+. Go to *Salesforce Set-up* by selecting the gear icon and selecting *Set-up*.
```

Comment:
clicking the gear icon

Reply:
Fixed. Thank you! I'll do a global review of the use of "select" in the ACM docs. At my last job the rule was never to use "click", and always to use another word such as "choose" or "select", and so the ACM docs were originally written that way. I'll re-write to follow the style guide.